sample_files/README.rst

```diff
@@ -0,0 +1,61 @@
+|License MIT| | |Build Status| | |Coveralls Status| | |Codecov Status| | |Code Climate|
+
+==================
+{{ Project Name }}
+==================
+
+This library does something, hopefully.
+
+Installation
+============
+
+* Install Python requirements ``pip install -r ./requirements``
+
+Setup
+=====
+
+* 
+
+Usage
+=====
+
+* `Read The API Documentation <https://laslabs.github.io/{{ Repo Slug }}>`_
+
+Known Issues / Road Map
+=======================
+
+-  Installation, setup, usage - in ReadMe
+
+Credits
+=======
+
+Images
+------
+
+* LasLabs: `Icon <https://repo.laslabs.com/projects/TEM/repos/odoo-module_template/browse/module_name/static/description/icon.svg?raw>`_.
+
+Contributors
+------------
+
+* My Name <myemail@example.com>
+
+Maintainer
+----------
+
+.. image:: https://laslabs.com/logo.png
+   :alt: LasLabs Inc.
+   :target: https://laslabs.com
+
+This module is maintained by LasLabs Inc.
+
+.. |Build Status| image:: https://api.travis-ci.org/LasLabs/{{ Repo Slug }}.svg?branch=master
+   :target: https://travis-ci.org/LasLabs/{{ Repo Slug }}
+.. |Coveralls Status| image:: https://coveralls.io/repos/LasLabs/{{ Repo Slug }}/badge.svg?branch=master
+   :target: https://coveralls.io/r/LasLabs/{{ Repo Slug }}?branch=master
+.. |Codecov Status| image:: https://codecov.io/gh/laslabs/{{ Repo Slug }}/branch/master/graph/badge.svg
```

Comment:
Fix LasLabs casing (old casing only applied to pre-existing links in some instances)